package.json

```diff
@@ -36,7 +36,7 @@
     "test:debug": "mocha debug test/runner.js --reporter spec"
   },
   "dependencies": {
-    "broccoli-caching-writer": "^0.5.4",
+    "broccoli-caching-writer": "^1.1.0",
```

Comment:
I didn't update, because the it uses the new `broccoli-plugin` and I wasn't exactly sure how to implement it here (plus had no time). I think it could be updated later, because other things have to be implemented, like a `build` method.